config/initializers/bugsnag.rb

```diff
@@ -1,3 +1,4 @@
 Bugsnag.configure do |config|
   config.api_key = ENV['BUGSNAG_API_KEY']
+  config.ignore_classes << ActiveRecord::RecordNotFound
```

Comment:
I'm not sure I want to ignore these in bugsnag long-term, because what shows on the UI is an error and we don't want to show the user errors- rather we want to show "not found" or similar. It is convenient for now tho :)